app-emulation/lxd/files/lxd.initd

```diff
@@ -0,0 +1,46 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+DAEMON=/usr/sbin/lxd
```

Comment:
this line should be defined in as the `command` parameter according to `openrc-run (8)`.

Reply:
@monsieurp I didn't write this file and I'm not familiar with writing init scripts. Do you know of an existing package that has proper init script?